firestore-tests/src/firestore.spec.ts

```diff
@@ -212,6 +216,66 @@ describe('Rounds', () => {
   });
 
 
+  describe('hostEvents', () => {
+    let round;
+
+    const hostPauseEvent = {
+      eventType: HostEventType.Pause,
+      timePeriod: 0
+    };
+
+    const hostPlayEvent = {
+      eventType: HostEventType.Play,
+      timePeriod: 12
+    };
+
+    beforeEach(async () => {
+      await addStudentToSession(admin, 'Linus', session.id, {
+        name: 'Linus Torvalds',
+        nestBarcode: 91
+      });
```

Comment:
This user isn't used in any tests, what is the point of adding it?

Reply:
ah, I planned on using the the student, but for got to. I can get rid of it.  Carol and bob are enough show that students can't read or add hostEvents